department-of-veterans-affairs/va.gov-cms
# VACMS-5141: Remove health from Vet Center Service content type name.

## Description

See #5141. 

## Testing done

Behat, manual

## Screenshots


## QA steps

As content admin
1. Go to /node/add
2. Note "Vet Center - Facility Service" content type name no longer has health in it.

### Definition of Done


### Select Team for PR review
